src/main/perf/TaskParser.java

```diff
@@ -70,6 +71,7 @@ public TaskParser(IndexState state,
     this.doStoredLoads = doStoredLoads;
     this.state = state;
     titleDVSort = new Sort(new SortField("titleDV", SortField.Type.STRING));
+    titleBDVSort = new Sort(new SortField("titleBDV", SortField.Type.STRING_VAL));
```

Comment:
@dsmiley this is how the benchmark is specifying to sort by BinaryDocValues. Also see FieldComparator.TermValComparator and its getLeafComparator and getBinaryDocValues

Reply:
Thanks for the pointer @msokolov !  It's pretty straight-forward but I was unfamiliar with this because Solr doesn't offer this option; at least not natively.